### DISCUSSION TITLE
All conflicts shown, how to disable?

### DISCUSSION THREAD

**Discussion by TMaekler** [bcssov/IronyModManager#446] [Q&A] [answered]
Hi. Looks like I messed up some settings of IMM. When using the conflict solver, it shows me EVERY conflict. It didn't before. Any ideas, about what I could have messed up?

**Comment by bcssov** [comment, marked as answer]
I'm not sure to what exactly are you referring to but it seems like:
1. You maybe turned on show game conflicts under `mod filters` which shows conflicts with vanilla game files
2. Didn't setup ignore rules filters

**Reply by TMaekler** [reply]
Yes, that's it. That whole list was selected... Thanks.
